lib/rubocop/cop/sorbet/rbi_versioning/valid_gem_version_annotations.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Sorbet
+      # Checks that gem versions in RBI annotations are properly formatted per the Bundler gem specification.
+      #
+      # @example
+      #   # bad
+      #   # @version > not a version number
+      #
+      #   # good
+      #   # @version = 1
+      #
+      #   # good
+      #   # @version > 1.2.3
+      #
+      #   # good
+      #   # @version <= 4.3-preview
+      #
+      class ValidGemVersionAnnotations < Base
```

Comment:
```suggestion
      class ValidGemVersionAnnotations < RuboCop::Cop::Base
```
(https://github.com/Shopify/rubocop-sorbet/pull/246)